graph/src/components/ethereum/adapter.rs

```diff
@@ -84,9 +86,50 @@ impl From<Error> for EthereumAdapterError {
     }
 }
 
-#[derive(Clone, Debug)]
+#[derive(Copy, Clone, Debug, PartialEq, Eq, Ord, PartialOrd, Hash)]
+enum LogFilterNode {
+    Contract(Address),
+    Event(EventSig),
+}
+
+/// Corresponds to an `eth_getLogs` call.
+#[derive(Clone)]
+pub struct EthGetLogsFilter {
+    pub contracts: Vec<Address>,
+    pub event_sigs: Vec<EventSig>,
+}
+
+impl fmt::Display for EthGetLogsFilter {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        if self.contracts.len() == 1 {
+            write!(
+                f,
+                "contract {}, {} events",
```

Comment:
I noticed in the logs that the contract address is ellipsized in the middle (e.g. `contract 0x1bfd…59ba`). Could we make logs contain the complete address so it's easier to copy and paste it?

Reply:
Ah yhea that's annoying.